main.js

```diff
@@ -0,0 +1,31 @@
+function Add(numbers = "") {
+    if (numbers.trim() == "") return 0;
+    let self = numbers.trim().split(',')
+    if (self.length > 2) {
+        throw new Error("Exceeded Numbers")
+    }
+    let sum = 0;
+
+    let negStack = []
+    
+    for (item of self) {
+        if (isNaN(item) === true) {
+            throw new Error("Wrong Numbers")
```

Comment:
We should parse the numbers and if the user pass strings return this error at the beginning